.github/workflows/release.yml

```diff
@@ -7,7 +7,7 @@ permissions:
   contents: write
 env:
   GH_ANNOTATION: true
-  DOCKER_REGISTRY: ghcr.io/linkerd
+  DOCKER_REGISTRY: cr.l5d.io/linkerd
```

Comment:
The `cr.l5d.io` domain works when pulling images, but I'm not sure if it'll work when pushing, because we still need to tag the image as `ghcr.io/linkerd/linkerd-smi:tag`. That's how it's setup in the release workflow in the linkerd2 repo...